Files/MultilingualResources/Files.ar.xlf

```diff
@@ -2575,7 +2575,19 @@
           <source>Open Storage Sense</source>
           <target state="new">Open Storage Sense</target>
         </trans-unit>
+        <trans-unit id="ToolTipDescriptionName" translate="yes" xml:space="preserve">
```

Comment:
Don't modify MultilingualResources files manually. These files will be regenerated after String en-US file changed. It is better to move space after `:` to the string.Format("{0}: {1}\n{2}: {3}\n{4}: {5}"